app/models/legal_aid_application.rb

```diff
@@ -157,6 +159,57 @@ def ccms_case_reference
     most_recent_ccms_submission.case_ccms_reference
   end
 
+  def add_default_scope_limitation!
+    if used_delegated_functions?
+      add_default_delegated_functions_scope_limitation!
+    else
+      add_default_substantive_scope_limitation!
+    end
+  end
+
+  def add_default_substantive_scope_limitation!
+    ApplicationScopeLimitation.create!(
+      legal_aid_application: self,
+      scope_limitation: default_substantive_scope_limitation_for_first_proceeding_type,
+      substantive: true
+    )
+  end
+
+  def add_default_delegated_functions_scope_limitation!
+    ApplicationScopeLimitation.create!(
+      legal_aid_application: self,
+      scope_limitation: default_delegated_functions_scope_limitation_for_first_proceeding_type,
+      substantive: false
+    )
+  end
+
+  def default_substantive_scope_limitation_for_first_proceeding_type
+    proceeding_types.first.default_substantive_scope_limitation
+  end
+
+  def default_delegated_functions_scope_limitation_for_first_proceeding_type
+    proceeding_types.first.default_delegated_functions_scope_limitation
```

Comment:
`proceeding_types.first` is used a lot in this PR.
I suggest creating and using this method:
```ruby
def proceeding_type
 proceeding_types.first
end
```

Reply:
I've created a method called lead_proceeding_type which is a concept we  need for injecting the application into CCMS, and will be something we need when we have multiple proceeding types per application